alembic/versions/4c9ae1c0db43_set_guestgroup_event_id_to_null_when_.py

```diff
@@ -1,7 +1,7 @@
 """Set GuestGroup event ID to null when their Event is deleted
 
 Revision ID: 4c9ae1c0db43
-Revises: 73b22ccbe472
+Revises: 61734fcb2e72
```

Comment:
This and the other change are just cosmetic, you can see they're inside a comment.